src/sections/Internship-gird/InternshipPage.style.js

```diff
@@ -52,50 +45,90 @@ const InternshipSectionWrapper = styled.div`
     }
 
     .opportunity-card {
-        display: inline-flex;
-        min-width: 16rem;
-        height: 5rem; 
-        background: rgba(247,247,247,1);
-        justify-content: center;
+        width: 20rem;
+        height: 14rem;
+        margin :auto;
+        padding: 0.5rem;
+        background: rgba(245,245,245,1);
+        text-align: center;
+        border-bottom: 5px solid ${props => props.theme.secondaryColor};
+        transition: all 0.2s;
+        transition-timing-function: ease-in-out;
+        h3{ 
+            font-weight: 500;
+        }
+        p{
+            margin: 0.5rem 0;
+            color: rgba(0, 0, 0, 0.8);
+        }
+        &:hover{
+            box-shadow: 0px 4px 20px 2px ${props => props.theme.shadowLightColor};
+        }
+    }
+
+    .full_time {
+        .opportunity-card {
+            height: 12rem;
+        }
+    }
+
+    @media screen and (max-width: 991px) {
+        .opportunity-card {
+            height: 12.5rem;
+        }
+    }
+
+    .partner-image {
+        height: 13rem;
+        margin-bottom: 0.25rem;
+
+        img {
+            width: 22rem;
+            height: inherit;
+            border-radius: 0.5rem;
+        }
+    }
+
+    .partners-card {
+        display: block;
+        width: max-content;
+        background: rgba(241,242,241,1);
         border-radius: 1rem;
+        text-align: center;
+        justify-content: space-between;
         transition: all 0.2s;
         transition-timing-function: ease-in-out;
         h5{
-            color: black; 
-            font-weight: 600;
-            align-self: center;
-            margin: auto 0.5rem;
-            transition: all 0.2s;
+            display: inline-flex;;
+            font-weight: 400;
+            line-height: 3.5rem;
+            vertical-align: middle;
```

Comment:
Ahhh....I think now it makes the text and the button not properly aligned vertically 😬....I know technically this is correct, but visually `top` was working 😆